mypyc/namegen.py

```diff
@@ -30,6 +40,9 @@ class NameGenerator:
       collides '___' with '.3_', but this is OK because names
       may not start with a digit.)
 
+    * Replace non-ASCII characters with their codepoints, such as '_e9'
+      for U+00E9.
```

Comment:
The downside here is that we'll get name collisions if `_e9` appears in a name. Can we come up with an encoding scheme that dodges this without uglifying normal names too much, like we did for `.` -> `___`. Would just using `___3[the unicode code point]_` work?